src/models/model-constants.js

```diff
@@ -73,12 +73,12 @@ const chedStatusDescriptions = {
 const displayClosedChedStatuses = ['CANCELLED', 'DELETED', 'REPLACED']
 
 const finalStateMappings = {
-  0: 'Released',
-  1: 'Cancelled',
-  2: 'Cancelled',
+  0: 'Cleared',
```

Comment:
According to:
https://eaflood.atlassian.net/wiki/spaces/ALVS/pages/5712052561/BTMS+FinalState+Field
The BTMS value is `Released`